code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -150,7 +150,7 @@
 	knowledge_data["gainFlavor"] = initial(knowledge.gain_text)
 	knowledge_data["cost"] = initial(knowledge.cost)
 	knowledge_data["disabled"] = (!done) && (initial(knowledge.cost) > knowledge_points)
-	knowledge_data["bgr"] = (path_to_ui_bgr[initial(knowledge.route)] || "side")
+	knowledge_data["bgr"] = (path_to_ui_bgr[GLOB.heretic_research_tree[knowledge][HKT_ROUTE]] || "side")
```

Comment:
Do we want to put `ui_bgr` as a var on the paths? Seems sensible

Reply:
yeah it does seem sensible i can add it to the metadata